spec/unit/generator/info_plist_file_spec.rb

```diff
@@ -74,10 +31,46 @@ module Pod
         'CFBundleVersion' => '${CURRENT_PROJECT_VERSION}',
         'NSPrincipalClass' => '',
       }
+
+      it 'sets the package type' do
+        generator = Generator::InfoPlistFile.new('1.0.0', Platform.new(:ios, '6.0'), :appl)
+        file = temporary_directory + 'Info.plist'
+        generator.save_as(file)
+        Xcodeproj::Plist.read_from_path(file).should == {
+          'CFBundleDevelopmentRegion' => 'en',
+          'CFBundleExecutable' => '${EXECUTABLE_NAME}',
+          'CFBundleIdentifier' => '${PRODUCT_BUNDLE_IDENTIFIER}',
+          'CFBundleInfoDictionaryVersion' => '6.0',
+          'CFBundleName' => '${PRODUCT_NAME}',
+          'CFBundlePackageType' => 'APPL',
+          'CFBundleShortVersionString' => '1.0.0',
+          'CFBundleSignature' => '????',
+          'CFBundleVersion' => '${CURRENT_PROJECT_VERSION}',
+          'NSPrincipalClass' => '',
+        }
+      end
+
+      it 'adds NSPrincipalClass for OSX platform' do
```

Comment:
new test that verifies that the Info.plist includes `NSApplication` as the value of `NSPrincipalClass` that is required by app targets for macOS.